database.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URI = 'sqlite:///library.db'
+
+engine = create_engine(
+    SQLALCHEMY_DATABASE_URI, connect_args={"check_same_thread": False}
```

Comment:
The `connect_args` parameter with `check_same_thread: False` is specific to SQLite and allows the use of the same connection across multiple threads. Ensure this is appropriate for your application's threading model.